pageserver/src/layered_repository/delta_layer.rs

```diff
@@ -339,6 +395,25 @@ impl DeltaLayer {
         Ok(delta_layer)
     }
 
+    fn open_book(&self) -> Result<(PathBuf, Book<File>)> {
```

Comment:
The 'PathBuf' return value is never used. Remove?

Reply:
Whoops, meant to use that in `load()`. Fixed now